content/SCALE/SCALEUIReference/Storage/Datasets/ImportDataScreenSCALE.md

```diff
@@ -22,5 +23,5 @@ For more information on importing data see [Importing Disks]({{< relref "Importi
 | **Destination Path** | Enter or use the <span class="material-icons">arrow_right</span> to the left of the <span class="material-icons">folder</span>**/mnt** to expand each level of the path until you reach the location where you want to import (mount) the data. Click on the dataset to select it and populate the path. |
 | **Import** | Starts the data import process. |
 
-{{< taglist tag="scaledatasets" limit="10" >}}
+{{< taglist tag="scaleimport" limit="10" title="Related Import Articles" >}}
```

Comment:
Delete line 26 as these article links are referenced in Line 27.

Reply:
The link to import articles includes the import pools article so this needs to remain as a taglist. :-)